src/components/OptionRow.js

```diff
@@ -212,6 +212,7 @@ class OptionRow extends Component {
                             accessibilityRole={CONST.ACCESSIBILITY_ROLE.BUTTON}
                             hoverDimmingValue={1}
                             hoverStyle={this.props.hoverStyle}
+                            needsOffscreenAlphaCompositing={this.props.option.icons.length >= 2}
```

Comment:
This line causes regression

Steps to reproduce:
1. FAB > Request money > Manual tab
2. Click currency symbol